### PR TITLE
solution_Catalog-grid

### PR DESCRIPTION
**Pull request by vladyslav-harkusha**
- [DEMO LINK](https://vladyslav-harkusha.github.io/layout_catalog_grid/)

### CODE REVIEW

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
-  margin: 0 auto;
+  $media:
+    488px 2,
+    768px 3,
+    1024px 4;
+  @each $viewport-width, $columns in $media {
+    @media(min-width: #{$viewport-width}) {
+      grid-template-columns: repeat(#{$columns}, 200px);
+    }
+  }
```
Comment (JHesher) [inline src/styles/_catalog.scss:14]: 🔥

**Review by JHesher — APPROVED** [review]

File: src/styles/_catalog.scss [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  $media:
+    488px 2,
+    768px 3,
+    1024px 4;
+  @each $viewport-width, $columns in $media {
+    @media(min-width: #{$viewport-width}) {
```
Comment (Elena-brv) [inline src/styles/_catalog.scss:11]:
```suggestion
    1024px 4;
    
  @each $viewport-width, $columns in $media {
    @media(min-width: #{$viewport-width}) {
```

**Review by Elena-brv — APPROVED** [review]
make cards look ok on small screens
image

File: src/styles/_catalog.scss (hunk 2 above)
Comment (vladyslav-harkusha) [reply src/styles/_catalog.scss]: +

**Pull request closed without merging**